code/Rounds/InProgressRound.cs

```diff
@@ -2,6 +2,9 @@
 {
     public class InProgressRound : BaseRound
     {
+        public override string RoundName => "Preround";
+        public override int RoundDuration => 100;
+        
```

Comment:
C'est une erreur de copié collé ou c'est normal que lui aussi s'appelle Preround ?
```suggestion
        public override string RoundName => "In progress round";
        public override int RoundDuration => 100;
        
```